### PR TITLE
Fix MSI uninstallation procedure tmp file locked

### LINKED ISSUE #6947: Error code 2318 during Windows uninstallation
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.1.0-alpha1 | Wazuh agent | Agent | Packages | Windows Server 2016 |

Hello team,

We've detected this error during Windows agent v4.1.0 uninstallation:

error_windows

"_The installer has encountered an unexpected error installing this package. This may indicate a problem with this package. The error code is 2318_"

# Steps to reproduce:

Register the agent to a manager. Then while the agent is running start the uninstallation process and proceeds normally

Tested package:  https://packages-dev.wazuh.com/pre-release/windows/wazuh-agent-4.1.0-1.msi

### PR DESCRIPTION
**Pull request by okynos**
|Related issue|
|---|
| Closes #6947 |

Hello team,

## Description

This PR aims to improve the uninstall procedure, we have changed the `StopWinService`, `StopOssecService` and `DeleteOssecService` to match the new CMD execution. This way to stop the process is more natural to Windows systems.
We have added a binary key that references the CMD executable.
We have changed the order of service execution inside the install sequence scheme.

## Tests

We have tested install and uninstall steps.
We have tried to reproduce the issue with this new change without success.

- Compilation without warnings in every supported platform
  - [x] Windows
- [x] Package installation
- [x] Package upgrade
- [x] Review logs syntax and correct language

- [x] Retrocompatibility with older Wazuh versions

Regards.

### CODE REVIEW

**Review by DFolchA — APPROVED** [review]
LGTM

**Review by DProvinciani — APPROVED** [review]
Looks good!

File: src/win32/wazuh-installer.wxs [hunk 1]
```diff
@@ -71,6 +71,7 @@
         <Property Id="ApplicationFolderName" Value="ossec-agent" />
         <CustomAction Id="LaunchApplication" BinaryKey="WixCA" DllEntry="WixShellExec" Impersonate="no" />
         <Binary Id="InstallerScripts" SourceFile="InstallerScripts.vbs" />
+        <Binary Id="CMDPath" SourceFile="C:\Windows\System32\cmd.exe" />
```
Comment (alberpilot) [inline src/win32/wazuh-installer.wxs:74]: As per conversations, the path can't be harcoded. @okynos will develop another approach.

**Review by alberpilot — CHANGES_REQUESTED** [review]
As per conversations, the path can't be harcoded. @okynos will develop another approach.

**Review by DFolchA — APPROVED** [review]

**Review by DProvinciani — APPROVED** [review]
Looks good!

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**